scripts/install_chrome_on_travis.py

```diff
@@ -52,7 +52,7 @@ def main(args=None):
 
     if not os.path.isfile(travis_chrome_path):
         os.chdir(os.path.join(home_directory, '.cache/TravisChrome/'))
-        python_utils.url_retrieve(
+        python_utils.url_retrieve(  # pylint:disable=arg-name-for-non-keyword-arg
             chrome_source_url, filename=os.path.basename(chrome_source_url))
         os.chdir(oppia_dir)
```

Comment:
We need to use filename here else backend tests are failing for this module, that's why I used pylint pragma to disable the warning. This is happening because we are using swap_with_checks() in the test file to swap url_retrieve with a mock function but we need the argument filename here because swap_with_checks() use *args and **kwargs and we must need argument names with it.

Reply:
Why don't we add a keyword argument in url_retrieve function? I'm not sure about the right way, but I posted my concern.

Reply:
Here we are using a position argument as a keyword argument `filename` but we don't use it then the backend tests are failing.

Reply:
Still not answered: Why don't we add a keyword argument in url_retrieve function? I'm not sure about the right way, but I posted my concern.

Reply:
Added the keyword argument